kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/streamable/AbstractKafkaTrackingTokenSerializeTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.extensions.kafka.eventhandling.consumer.streamable;
+
+import org.apache.kafka.common.TopicPartition;
+import org.axonframework.eventhandling.ReplayToken;
+import org.axonframework.eventhandling.TrackingToken;
+import org.axonframework.serialization.SerializedObject;
+import org.axonframework.serialization.Serializer;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static java.util.Collections.singletonMap;
+import static org.axonframework.extensions.kafka.eventhandling.consumer.streamable.KafkaTrackingToken.newInstance;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Abstract tests serialization the {@link KafkaTrackingToken}.
+ *
+ * @author leechedan
+ * @author leechedan
+ */
+public abstract class AbstractKafkaTrackingTokenSerializeTest {
```

Comment:
Although this works, it is somewhat out of line with how the other serialization tests are set up for the framework and its extensions.
The `GlobalSequenceTrackingTokenSerializationTest` in Axon Framework (which you can find [here](https://github.com/AxonFramework/AxonFramework/blob/master/messaging/src/test/java/org/axonframework/eventhandling/GlobalSequenceTrackingTokenSerializationTest.java)) is an example of a parameterized test.

So, if you could please change the constructed tests to the parameterized format, that would be great.